buildAndRunSite.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-jekyll serve --incremental --watch --config _config.yml,_config_en.yml 
\ No newline at end of file
+jekyll serve --incremental --watch --config _config_default.yml,_config_en.yml
```

Comment:
Hey mani, I think what you need here is `--config _config.yml _config_default.yml` That will tell jekyll to serve only the english the language when using this script. How is it now, I am not sure if it will work, _config_default has only the language specification